### PR TITLE
Fix CI Maven build error

### PR DESCRIPTION
**Pull request by typhoonzero**
Seems https://github.com/travis-ci/travis-ci/issues/6593#issuecomment-417752102 works.

### CODE REVIEW

**Comment by tonyyang-svail** [comment]
@typhoonzero NB.

File: .travis.yml [hunk 1]
```diff
@@ -14,6 +14,7 @@ branches:
   - "/^v\\d+\\.\\d+(\\.\\d+)?(-\\S*)?$/"
 service:
   - docker
```
Comment (tonyyang-svail) [inline .travis.yml:16]: Could you add a comment on this change? For example,

```
# NOTE(typhoonzero): `sudo: required` fixes "maven build fails with ACCESS DENIED".
# ref: https://github.com/travis-ci/travis-ci/issues/6593#issuecomment-417752102
```

**Review by tonyyang-svail — DISMISSED** [review]
Thanks for the fix!

**Review by tonyyang-svail — COMMENTED** [review]
The CI failed on Maven build again, so maybe this PR won't solve the issue.

**Comment by typhoonzero** [comment]
Closing due to https://github.com/sql-machine-learning/sqlflow/pull/1667

**Pull request closed without merging**